trailforks-export.js

```diff
@@ -10,17 +10,17 @@ function downloadString(filename, text) {
 }
 
 //checks for problems, alerts user
-function errCheck(pageString) {
+function errCheck(pageStr) {
```

Comment:
Assuming you changed `pageString` to `pageStr` to avoid potential conflict?

Reply:
Exactly 👌